### PR TITLE
added bosnian translation for txadmin

### PR DESCRIPTION
**Pull request by Arm-0001**

### CODE REVIEW

File: locale/bh.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        "humanizer_language": "bh"
+    },
+    "restarter": {
+        "start_timeout": "Server nije uspio započeti na vrijeme",
+        "crash_detected": "Server se zaustavio",
+        "partial_crash_warn": "Zbog djelomičnog pada, Server će se ponovo pokrenuti za 1 minutu. Isključiti te se sada.",
+        "partial_crash_warn_discord": "Zbog djelomičnog pada, **%{servername}** će se restartovati",
+        "schedule_reason": "restart je zakazan za %{time}",
+        "schedule_warn": "Ovaj server ce se restartovati za %{smart_count} minute. Molim vas se isključitite. |||| Ovaj server ce se restartovati za %{smart_count} minute.",
+        "schedule_warn_discord": "**%{servername}** restarta se za %{smart_count} minuta. |||| **%{servername}** restarta se za %{smart_count} minuta."
```
Comment (tabarra) [inline locale/bh.json]: is the plural vs singular correct in here? have you tested it in-game?

Comment (Arm-0001) [reply locale/bh.json]: yes they are and it isnt tested

**Comment by tabarra** [comment]
Tested?

**Comment by Arm-0001** [comment]
> Tested?

no im trying to figure out how to use verbose

**Comment by Arm-0001** [comment]
> Tested?

can you test pls 👉👈

**Comment by PLOKMJNB** [comment]
your the one that knows bosnian

**Comment by Arm-0001** [comment]
> your the one that knows bosnian

teach me how to use verbose :D <3

**Comment by PLOKMJNB** [comment]
>teach me how to use verbose :D <3

https://github.com/tabarra/txAdmin/blob/master/README.md#convars

**Comment by Arm-0001** [comment]
yes ik how to turn it on but how do you use it

**Review by tabarra — CHANGES_REQUESTED** [review]
The international language code for Bosnian is `bs` and not `bh`.  
Also The humanize-duration library does not support bosnian, please set `humanizer_language` to `en`. Would do myself but you didn't enable the option to let me edit it... 🤷   
https://www.science.co.il/language/Locale-codes.php  
https://www.npmjs.com/package/humanize-duration

**Comment by tabarra** [comment]
Actually I will just merge manually.

**Pull request closed without merging**